File: Modul 6/Bahasa Python (modul 6)/PRAK602-2310817320015-AlfisyahRinaFajriati.py
def hitung_jumlah_zetsu_putih(ruangan, zetsu_putih):
    hasil = []

    for i in range(ruangan):
        jumlah_pembelahan = zetsu_putih[i] * (i + 1)
        hasil.append(jumlah_pembelahan)

    return hasil

ruangan = int(input(""))
zetsu_putih = list(map(int, input("").split()))

hasil = hitung_jumlah_zetsu_putih(ruangan, zetsu_putih) 

for i, jumlah in enumerate(hasil):
    print(" ".join(map(str, hasil))) 

File: Modul 6/Bahasa C (modul 6)/PRAK602-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
int main() {
    
    int ruangan, i;
    scanf("%d", &ruangan);

    int zetsu_putih[ruangan];
    for (i = 0; i < ruangan; i++) {
        scanf("%d", &zetsu_putih[i]);
    }

    for (i = 0; i < ruangan; i++) {
        int hasil = zetsu_putih[i] * (i + 1);
        printf("%4d", hasil);
    }

    return 0;
}
